lib/inject.rb

```diff
@@ -0,0 +1,24 @@
+class Array
+
+  def double_inject(initial=nil, &block)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for double_inject is too high. [21.02/15]